services/dumper.js

```diff
@@ -144,7 +144,11 @@ function Dumper(config) {
       const expectedConventionalColumnName = underscored ? _.snakeCase(field.name) : field.name;
       const nameColumnUnconventional = field.nameColumn !== expectedConventionalColumnName
         || (underscored && /[1-9]/g.test(field.name));
-      return { ...field, nameColumnUnconventional };
+      let ref;
+      if (field.ref) {
+        ref = getModelNameFromTableName(field.ref);
+      }
+      return { ...field, ref, nameColumnUnconventional };
```

Comment:
Nice job, you haven't mutated the original field! On a side note, just because I prefer anything over `let` (but you could consider this as a kind of madness), I would have preferred something like:

```javascript
return {
  ...field,
  ref: field.ref && getModelNameFromTableName(field.ref), // or a ternary operator
  nameColumnUnconventional,
};
```

But it is an opinion (I may be wrong) that you are free to ignore! (I just wanted to leave a comment in your PR)